core/src/main/java/com/graphhopper/reader/osm/OSMReader.java

```diff
@@ -252,12 +252,12 @@ protected void setArtificialWayTags(PointList pointList, ReaderWay way, double d
         // special handling for countries: since they are built-in with GraphHopper they are always fed to the EncodingManager
         Country country = Country.MISSING;
         for (CustomArea customArea : customAreas) {
-            Object countryCode = customArea.getProperties().get("ISO3166-1:alpha3");
-            if (countryCode == null)
+            Object area3 = customArea.getProperties().get(Country.JSON_AREA3);
+            if (area3 == null)
                 continue;
             if (country != Country.MISSING)
-                LOGGER.warn("Multiple countries found for way {}: {}, {}", way.getId(), country, countryCode);
-            country = Country.valueOf(countryCode.toString());
+                LOGGER.warn("Multiple countries found for way {}: {}, {}", way.getId(), country, area3);
+            country = Country.valueOf((String) area3);
```

Comment:
Now we have alpha2 and area3 :D Maybe rename the constant to `Country.ALPHA3` or `Country.ISO_ALPHA3`? Its value is no longer the name of the property that is used for the two letter country code in the country json file, so the `JSON_` prefix does not really make sense?